Changes

```diff
@@ -8,16 +8,18 @@ Next version (4.04.1):
 
 ### Bug fixes
 
-- PR#7405, GPR#903: s390x: Fix address of caml_raise_exn in native dynlink modules
-  (Richard Jones, review by Xavier Leroy)
+- PR#7405, GPR#903: s390x: Fix address of caml_raise_exn in native
+  dynlink modules (Richard Jones, review by Xavier Leroy)
```

Comment:
I guess your editor did this? This line had slipped through at 82 columns, but the credits should remain on a line on their own